doc/libpmemobj.3.md

```diff
@@ -2205,6 +2205,10 @@ goes to stderr.
 Setting the environment variable **PMEMOBJ_LOG_LEVEL** has no effect on the non-debug version of **libpmemobj**.
 See also **libpmem**(3) to get information about other environment variables affecting **libpmemobj** behavior.
 
+**libpmemobj** by default uses up to 1024 lanes (isolated paths of execution)
```

Comment:
This is not descriptive enough. After reading this, and having no other knowledge, I would still be unable to say what this variable changes.
The usage of the word "isolation" could, for some people, bring "sandboxing" to mind.

I'd suggest to say that this determines the maximum number of parallel transactions/allocations in the library. And that there's usually no reason to set this to anything different than default (other than debugging).